lib/takeaway_interface.rb

```diff
@@ -0,0 +1,40 @@
+require_relative "menu"
+require_relative "summary"
+require_relative "text"
+require_relative "order"
+
+module TakewayInterface
```

Comment:
why build a class in a module? Doesn't that give the module context and stop it being a module?

Reply:
to be honest, I found it pretty to write it in a Module, but it does not have any use at all, I wanted to remove it but I forgot